docs/src/standalone/pages/vegetation/canopy_energy/canopy_energy.md

```diff
@@ -0,0 +1,125 @@
+# Canopy Energy Balance (Bigleaf)
+
+This page documents the **canopy energy** module used in ClimaLand’s vegetation standalone runs.
+The ClimaLand code can be found [here](https://github.com/CliMA/ClimaLand.jl/blob/main/src/standalone/Vegetation/canopy_energy.jl).
+
+---
+
+## Prognostic canopy temperature
+
+The canopy temperature is modeled prognostically. Its time tendency is given by
+
+```math
+\frac{dT_c}{dt} =
+-\frac{
+    -LW_n - SW_n + \mathrm{SHF} + \mathrm{LHF} - F_\mathrm{roots}
+}{
+    a_c \cdot \max(A_\mathrm{leaf} + A_\mathrm{stem}, \epsilon)
+},
+```
+
+where
+
+ $T_{c}$ is the canopy temperature (K),
+
+ $LW_{n}$ is the net long-wave radiation (W m$^{-2}$),
+
+ $SW_{n}$ is the net short-wave radiation (W m$^{-2}$),
+
+ $\mathrm{SHF}$ is the sensible heat flux (W m$^{-2}$),
+
+ $\mathrm{LHF}$ is the latent heat flux (W m$^{-2}$),
```

Comment:
Could we call it LE and H like the general flux tower/energy flux community consensus?